modules/webgl/src/classes/vertex-array-object.js

```diff
@@ -68,6 +68,7 @@ export default class VertexArrayObject extends Resource {
     this.hasVertexArrays = VertexArrayObject.isSupported(gl);
     this.buffer = null;
     this.bufferValue = null;
+    this.isDefaultArray = opts.isDefaultArray !== undefined ? opts.isDefaultArray : false;
```

Comment:
Drop `opts.isDefaultArray ` and just initialize like this?

```
this.isDefaultArray = !this.handle;
```

Reply:
I kind of prefer making this explicit in case there are ever any other semantics for a null handle. But it could be simplified to `this.isDefaultArray = opts.isDefaultArray || false`